html/objects/_flag.scss

```diff
@@ -102,18 +102,12 @@
 
 /// Vertically align content to the middle.
 .sprk-o-Flag--middle {
-  > .sprk-o-Flag__body,
-  > .sprk-o-Flag__figure {
-    vertical-align: middle;
-  }
+  align-items: center;
```

Comment:
Talked in chat, this should be updated because it centers the children horizontally on mobile because of the flexbox cross axis change from row to column.